use_cases/sending_amp_html_content.py

```diff
@@ -0,0 +1,99 @@
+# Sending AMP-HTML Email
+
+
+Currently, we require AMP HTML and any one of HTML or Plain Text content for improved deliverability or fallback for AMP HTML Email for supporting older clients and showing alternate content after 30 days. In some cases, only HTML may be available. The below example shows how to obtain the Plain Text equivalent of the HTML content.
```

Comment:
This description does not match the code below. Maybe something like: "Following is an example on how to send an AMP HTML Email."